addons/project/data/mail_data.xml

```diff
@@ -81,9 +82,10 @@
         <field name="name">Task Rating</field>
         <field name="sequence">14</field>
         <field name="res_model">project.project</field>
-        <field name="default" eval="True"/>
+        <field name="default" eval="False"/>
         <field name="parent_id" ref="mt_task_rating"/>
         <field name="relation_field">project_id</field>
+        <field name="hidden" eval="True"/>
```

Comment:
We'll need a migration for that, as the file is noupdate.
But I'm discussing with aju the necessity to fix in 15.0.
If checking the subtypes as no consequence, we should do it in master IMO.
I'll keep you in touch :slightly_smiling_face:

Reply:
Okay :slightly_smiling_face:

Reply:
After discussing it with aju, it is better to do it in master :pray: